src/sentry/incidents/models.py

```diff
@@ -417,6 +418,29 @@ class Meta:
         unique_together = (("alert_rule", "project"),)
 
 
+@region_silo_only_model
+class AlertRuleProject(Model):
```

Comment:
Just one extra question regarding this - Is there any chance that release monitoring alerts should actually be at the org level? So we'd just enable it for the entire org, and then we'd create subscriptions when releases are created/deployed in any project?

Reply:
I don't think so 🤔 
ActivatedAlertRules would be on a per-project basis.
We _could_ also potentially allow for full org activated rules, but that's a secondary use case atm

Reply:
Sounds good, just wanted to check!